CHANGELOG.md

```diff
@@ -120,6 +185,7 @@
 - Add dependabot [\#227](https://github.com/AbsaOSS/k8gb/pull/227) ([donovanmuller](https://github.com/donovanmuller))
 - bump AbsaOSS/k3d-action to version v1.0.0 [\#226](https://github.com/AbsaOSS/k8gb/pull/226) ([kuritka](https://github.com/kuritka))
 - Changelog for v0.7.2 [\#225](https://github.com/AbsaOSS/k8gb/pull/225) ([ytsarev](https://github.com/ytsarev))
+- k3d migration [\#218](https://github.com/AbsaOSS/k8gb/pull/218) ([kuritka](https://github.com/kuritka))
```

Comment:
Why this is added?

Reply:
I better run `github_changelog_generator` locally and amend.

Reply:
Amended